execution_workflows/GETUTR/nextflow.config

```diff
@@ -0,0 +1,141 @@
+/*
+ * -------------------------------------------------
+ *  APAeval Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ */
+
+// Global default params, used in configs
+params {
+
+  // Options: Mandatory
+  input                      = ''
+
```

Comment:
as commented above, can we specify the gtf input file here as a flag as it is the same across the samples from each run

Reply:
Thank you Yuk Kei! I will implement it this afternoon and post an update here, thanks again for reviewing and write soon

Reply:
Are we missing a gtf param here still?